### ISSUE TITLE
provider failed - Unable to find R interpreter

### ISSUE THREAD

**Issue by IsmailM** [MikhailArkhipov/vscode-r#48]
I see the following error:

```bash
[2020-11-17 09:27:09.406] [exthost] [error] [Mikhail-Arkhipov.r] provider FAILED
[2020-11-17 09:27:09.407] [exthost] [error] Error: name must not be falsy
    at Function.validate (/home/ucbtmog/.vscode-server/bin/e5a624b788d92b8d34d1392e4c4d9789406efe8f/out/vs/server/remoteExtensionHostProcess.js:548:978)
    at new C (/home/ucbtmog/.vscode-server/bin/e5a624b788d92b8d34d1392e4c4d9789406efe8f/out/vs/server/remoteExtensionHostProcess.js:548:927)
    at new C (/home/ucbtmog/.vscode-server/bin/e5a624b788d92b8d34d1392e4c4d9789406efe8f/out/vs/server/remoteExtensionHostProcess.js:535:436)
    at asSymbolInformation (/home/ucbtmog/.vscode-server/extensions/mikhail-arkhipov.r-0.0.15/node_modules/vscode-languageclient/lib/protocolConverter.js:449:22)
    at /home/ucbtmog/.vscode-server/extensions/mikhail-arkhipov.r-0.0.15/node_modules/vscode-languageclient/lib/protocolConverter.js:429:42
    at Array.map (<anonymous>)
    at Object.asSymbolInformations (/home/ucbtmog/.vscode-server/extensions/mikhail-arkhipov.r-0.0.15/node_modules/vscode-languageclient/lib/protocolConverter.js:429:23)
    at /home/ucbtmog/.vscode-server/extensions/mikhail-arkhipov.r-0.0.15/node_modules/vscode-languageclient/lib/client.js:1088:70
```

I am running this over a vscode SSH setup with the following in my vscode settings.json:

```json
{
	"r.interpreterPath": "/home/ucbtmog/.asdf/installs/R/4.0.2/lib64/R/bin/R"
}``` 

Any ideas on what the issue is?

**Comment by adroste** [comment]
Also happens on my setup:

macOS 11.1
VS Code 1.53.0
R/Radian installed via brew
Interpreter Path set to `/usr/local/Cellar/r/4.0.3/bin/R`
terminal path to `/usr/local/bin/radian`

Holds for v0.0.15 (with .NET Core 3.1 LTS) as well as v0.0.16 (with .NET 5.0)

**Comment by MikhailArkhipov** [comment]
@adroste - what is brew command for radian? I tried via pip3 on 11.2, have some issues per https://github.com/randy3k/radian/issues/259

**Comment by MikhailArkhipov** [comment]
0.0.17 has been published

**Comment by adroste** [comment]
@MikhailArkhipov I think I installed radian via pip, sorry.
But radian should not be the problem. It also does not work without specifying radian as terminal.

On 0.0.18 same problem as before. I just cant get it to work.
```
[2021-02-18 16:18:47.954] [exthost] [error] [Mikhail-Arkhipov.r] provider FAILED
[2021-02-18 16:18:47.954] [exthost] [error] Error: name must not be falsy
	at Function.validate (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:80:32893)
	at new o (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:80:32842)
	at new o (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:80:19977)
	at asSymbolInformation (/Users/alexander/.vscode/extensions/mikhail-arkhipov.r-0.0.18/node_modules/vscode-languageclient/lib/common/protocolConverter.js:508:22)
	at /Users/alexander/.vscode/extensions/mikhail-arkhipov.r-0.0.18/node_modules/vscode-languageclient/lib/common/protocolConverter.js:476:42
	at Array.map (<anonymous>)
	at Object.asSymbolInformations (/Users/alexander/.vscode/extensions/mikhail-arkhipov.r-0.0.18/node_modules/vscode-languageclient/lib/common/protocolConverter.js:476:23)
	at /Users/alexander/.vscode/extensions/mikhail-arkhipov.r-0.0.18/node_modules/vscode-languageclient/lib/common/client.js:1111:70
```

```
[2021-02-18 16:18:55.677] [exthost] [error] [Mikhail-Arkhipov.r] provider FAILED
[2021-02-18 16:18:55.677] [exthost] [error] TypeError: Cannot read property 'kind' of undefined
	at asHoverContent (/Users/alexander/.vscode/extensions/mikhail-arkhipov.r-0.0.18/node_modules/vscode-languageclient/lib/common/protocolConverter.js:151:27)
	at asHover (/Users/alexander/.vscode/extensions/mikhail-arkhipov.r-0.0.18/node_modules/vscode-languageclient/lib/common/protocolConverter.js:191:31)
```

**Comment by MikhailArkhipov** [comment]
This is useful. Looks like document symbol provider was sending bad data. It never should have been active anyway. However, I implemented it properly in 0.0.19.

**Comment by MikhailArkhipov** [comment]
Feel free to reopen if you still see it in 0.22+
